pkg/downloader/manager.go

```diff
@@ -537,12 +544,20 @@ func tarFromLocalDir(chartpath string, name string, repo string, version string)
 		return "", fmt.Errorf("wrong format: chart %s repository %s", name, repo)
 	}
 
-	origPath, err := filepath.Abs(strings.TrimPrefix(repo, "file://"))
-	if err != nil {
-		return "", err
+	p := strings.TrimPrefix(repo, "file://")
```

Comment:
This code is repeated three times and has more logic to it than before, could this be abstracted into a common function to avoid duplication?